trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -91,9 +91,14 @@ const submitDealAfterUkefIds = async (dealId, dealType, checker) => {
       || mappedDeal.submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIA) {
       const updatedDealWithTasks = await createDealTasks(updatedDealWithCreateEstore);
 
+      /**
+       * Current requirement only allows AIN & MIN deals to be send to ACBS
+       */
       const updatedDeal = await api.updateDeal(dealId, updatedDealWithTasks);
-
-      await dealController.submitACBSIfAllPartiesHaveUrn(dealId);
+      if (mappedDeal.submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.AIN
```

Comment:
should we be checking the dea`mappedDeal.deallType` instead of submissionType? This would still get called for a GEF deal

Reply:
dealType will be either GEF or BSS/ECWS. I am looking for whether the deal's submission type is either AIN/MIN and etc.

Reply:
Sorry, think I misread the comment in the code 👍